examples/v1beta1/taskruns/entrypoint-resolution.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: tekton.dev/v1beta1
+kind: TaskRun
+metadata:
+  name: entrypoint-resolution
+spec:
+  taskSpec:
+    steps:
+    # Multi-arch image with no command defined. We should look up the command
+    # for each platform-specific image and pass it to the Pod, which selects
+    # the right command at runtime based on the node's runtime platform.
+    - image: gcr.io/tekton-nightly/github.com/tektoncd/pipeline/cmd/nop
+
+    # Single-platform image with no command defined, but with args. We'll look
+    # up the commands and pass it to the entrypoint binary via env var, then
+    # append the specified args.
+    - image: ubuntu
```

Comment:
Shouldn't it be using `amd64/ubuntu` if, according to the comment, it's a "single-platform" command ?

Reply:
(or the comment is wrong)

Reply:
Nice catch, the comment was wrong. 👍